packages/rsk3-utils/tests/utils.test.js

```diff
@@ -691,4 +694,44 @@ describe('UtilsTest', () => {
         expect(() => jsonInterfaceMethodToString(undefined)).toThrow();
         expect(() => jsonInterfaceMethodToString({name: 'name'})).toThrow();
     });
+
+    it('calls to getAddress and returns de expected value', () => {
+        const tests = [
+            {
+                value: 'Kxj9x4G2Uvw8CbngATxGZLY4K2EFkjty7mumLabJQEhHw9ZLyZTs',
+                expected: '36eb6a1d1720a6037d0680b8b357fc661a24bfdc'
+            }
+        ];
```

Comment:
Having an array of tests and then looping over them seems redundant if there is only going to be one test case.

To improve this I suggest having at least one "happy path" test case, which is what you already have here, and at least one "failure path" test case

For the failure path test cases, the function being called is expected to error. For example, if your provide it with an invalid input `value`.

In the implementations for these functions being tested, I currently do not see any validation logic, so perhaps the inclusion of these test cases will make for a more robust implementation.